ansible/playbooks/service/ghostunnel.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright (C) 2021 Pedro Luis Lopez <pedroluis.lopezsanchez@gmail.com>
+# Copyright (C) 2021 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Manage Ghostunnel service
+  collections: [ 'debops.debops' ]
```

Comment:
You might want to include all collections here (`debops.debops`, `debops.roles01`, `debops.roles02`, `debops.roles03`) since some roles might not be present in the main collection. For example, the `golang` role is included in the `debops.roles01` collection.

Reply:
Done!